packages/opentelemetry-sdk-trace-base/test/common/BasicTracerProvider.test.ts

```diff
@@ -92,19 +93,35 @@ describe('BasicTracerProvider', () => {
         },
       }).getTracer('default');
       assert.deepStrictEqual(tracer.getSpanLimits(), {
+        attributeValueLengthLimit: Infinity,
         attributeCountLimit: 100,
         eventCountLimit: 128,
         linkCountLimit: 128,
       });
     });
 
+    it('should construct an instance with customized attributeValueLengthLimit span limits', () => {
```

Comment:
Yes, I agree. It's adds better readability also 👍 

I followed existing code test cases pattern since its my first contribution to open source

I will make changes here

Reply:
@obecny I made test cases as you mentioned.